tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -28,9 +28,10 @@
     ip -6 addr add 2001:DB8::1/32 dev testbr
 
     if grep 'release 6' /etc/redhat-release; then
-        # We need radvd only in rhel6
+        # We need bridge-utils and radvd only in rhel6
         if ! rpm -q --quiet radvd; then yum -y install radvd; fi
-        
+        if ! rpm -q --quiet bridge-utils; then yum -y install bridge-utils; fi
```

Comment:
if you install stuff in the test setup, then you will not properly test whether the role installs the package itself.
Also, isn't it better to use the `package` module for this instead of shell commands, conditions, etc.?

Reply:
I need to install it just for RHEL6, not the others, how should I do that?

Reply:
I need setup to be deployed to mimic complex network setup.. few ethernets with ipv4/ipv6 addresses and dhcp.. I don't want to use eth0 for this for various reasons.

Reply:
This would allow to install packages depending on the distribution version:

```yaml
- name: Install dependencies
  package:
    name:
      - bridge-utils
      - radvd
 when:
   - ansible_distribution_major_version == '6'
```